app/controllers/admin/banners_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Admin
+  class BannersController < BaseController
+    layout 'admin'
+
+    def index
+      @banners = Banner.where('expires_at > ?', DateTime.now)
+    end
+
+    def new; end
+
+    def create
+      handle_with(BannersManage,
+        success: lambda do
+          flash[:success] = 'Banner created'
+          redirect_to admin_banners_path
+        end,
+        failure: lambda do
+          flash[:alert] = 'Error in saving. Please try again.'
```

Comment:
This error message is not too explicit. Are the fields that have errors highlighted at least when this happens?

Reply:
Fixed, as we discussed, just by adding `render :edit`.